src/languages/alma-woocommerce-gateway.pot

```diff
@@ -25,232 +25,248 @@ msgstr ""
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:157
+#: includes/class-alma-wc-admin-form.php:158
 msgid "Enable monthly payments with Alma"
 msgstr ""
 
 #. translators: %s Alma security URL
-#: includes/class-alma-wc-admin-form.php:178
+#: includes/class-alma-wc-admin-form.php:179
 #, php-format
 msgid "You can find your API keys on <a href=\"%s\" target=\"_blank\">your Alma dashboard</a>"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:181
+#: includes/class-alma-wc-admin-form.php:182
 msgid "Live API key"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:185
+#: includes/class-alma-wc-admin-form.php:186
 msgid "Test API key"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:189
+#: includes/class-alma-wc-admin-form.php:190
 msgid "API Mode"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:191
+#: includes/class-alma-wc-admin-form.php:192
 msgid ""
 "Use <b>Test</b> mode until you are ready to take real orders with Alma<br>In Test mode, only admins can see Alma on cart/checkout "
 "pages."
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:194
+#: includes/class-alma-wc-admin-form.php:195
 msgid "Test"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:195
+#: includes/class-alma-wc-admin-form.php:196
 msgid "Live"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:212
+#: includes/class-alma-wc-admin-form.php:213
 msgid "→ Fee plans configuration"
 msgstr ""
 
 #. translators: %s: Alma conditions URL
-#: includes/class-alma-wc-admin-form.php:219
+#: includes/class-alma-wc-admin-form.php:220
 #, php-format
 msgid "⚠ There is no fee plan allowed in your <a href=\"%s\" target=\"_blank\">Alma dashboard</a>."
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:235
+#: includes/class-alma-wc-admin-form.php:236
 msgid "Select a fee plan to update"
 msgstr ""
 
 #. translators: %s: Alma conditions URL
-#: includes/class-alma-wc-admin-form.php:238
+#: includes/class-alma-wc-admin-form.php:239
 #, php-format
 msgid ""
 "Choose which fee plan you want to modify<br>(only your <a href=\"%s\" target=\"_blank\">Alma dashboard</a> available fee plans are "
 "shown here)."
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:257
+#: includes/class-alma-wc-admin-form.php:258
 msgid "→ General configuration"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:261
-msgid "Title"
+#: includes/class-alma-wc-admin-form.php:262
+msgid "<p style=\"font-weight:normal;\">Edit the text displayed when choosing the payment method in your checkout.</p>"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:263
-msgid "This controls the payment method name which the user sees during checkout."
+#: includes/class-alma-wc-admin-form.php:267
+msgid "<p style=\"font-size:1.15em;\">Payments in 2, 3 and 4 installments:</p>"
 msgstr ""
 
-#: includes/class-alma-wc-admin-form.php:268
-msgid "Description"
+#: includes/class-alma-wc-admin-form.php:271
+msgid "<p style=\"font-size:1.15em;\">Deferred Payments:</p>"
```

Comment:
Do not use HTML in translations

use `sprintf('<p style"font-size:1.15em;">%s</p>', __("Deferred Payments:"))` instead